### ISSUE TITLE
Docs for documentation tests should mention where doctests are picked up from (e.g. not from files in tests/*)

### ISSUE THREAD

**Issue by robinst** [rust-lang/rust#79587]
(@GuillaumeGomez asked me to open an issue here, see his comment: https://github.com/GuillaumeGomez/doc-comment/issues/25#issuecomment-734745278)

The docs for documentation tests (https://doc.rust-lang.org/rustdoc/documentation-tests.html) don't currently mention in which files doctests can be put. E.g. they work if put into `src/*` files, but not in integration test files in `tests/*`. It would be good if that was mentioned there.

**Comment by GuillaumeGomez** [comment]
Actually, this should better be in the cargo book instead since it's a cargo-linked issue here.

**Comment by robinst** [comment]
Good point, maybe add a note here instead: https://doc.rust-lang.org/cargo/commands/cargo-test.html

> Documentation tests are also run by default, which is handled by rustdoc. It extracts code samples from documentation comments and executes them. See the rustdoc book for more information on writing doc tests.

**Comment by jyn514** [comment]
@ehuss could you please move this to rust-lang/cargo?
